source/reference/parameters.txt

```diff
@@ -558,6 +558,13 @@ General Parameters
    upgrade and then gradually resolve these indexing issues, you can
    use :parameter:`failIndexKeyTooLong` to disable this behavior.
 
+   .. important::
+
+      Setting :parameter:`failIndexKeyTooLong` to ``false`` is
+      intended to be a temporary workaround. Leaving
```

Comment:
Nit: Should we explicitly state what problem is solved by the workaround?

Reply:
I think it's adequately explained in the paragraphs above, but if you think the explanation should go in the note as well I can add it there.